beta/src/pages/learn/javascript-in-jsx-with-curly-braces.md

```diff
@@ -1,25 +1,24 @@
 ---
-title: JavaScript in JSX with Curly Braces
+title: घुंघराले ब्रेसिज़ के साथ JSX में JavaScript
 ---
 
 <Intro>
 
-JSX lets you write HTML-like markup inside a JavaScript file, keeping rendering logic and content in the same place. Sometimes you will want to add a little JavaScript logic or reference a dynamic property inside that markup. In this situation, you can use curly braces in your JSX to open a window to JavaScript.
-
+JSX आपको तर्क और सामग्री को एक ही स्थान पर प्रस्तुत करते हुए, JavaScript फ़ाइल के अंदर HTML जैसा मार्कअप लिखने देता है। कभी-कभी आप थोड़ा JavaScript तर्क जोड़ना चाहेंगे या उस मार्कअप के अंदर एक गतिशील संपत्ति का संदर्भ देना चाहेंगे। इस स्थिति में, आप JavaScript के लिए एक विंडो खोलने के लिए अपने JSX में घुंघराले ब्रेसिज़ का उपयोग कर सकते हैं।
 </Intro>
 
 <YouWillLearn>
 
-* How to pass strings with quotes
-* How to reference a JavaScript variable inside JSX with curly braces
-* How to call a JavaScript function inside JSX with curly braces
-* How to use a JavaScript object inside JSX with curly braces
+* उद्धरणों के साथ strings कैसे पास करें
+* घुंघराले ब्रेसिज़ के साथ JSX के अंदर एक JavaScript चर का संदर्भ कैसे लें
+* घुंघराले ब्रेसिज़ के साथ JSX के अंदर JavaScript फ़ंक्शन को कैसे कॉल करें
+* घुंघराले ब्रेसिज़ के साथ JSX के अंदर JavaScript ऑब्जेक्ट का उपयोग कैसे करें
 
 </YouWillLearn>
 
-## Passing strings with quotes {/*passing-strings-with-quotes*/}
+## उद्धरणों के साथ strings पास करना {/*उद्धरणों-के-साथ-strings-पास-करना*/}
```

Comment:
`{/*उद्धरणों-के-साथ-strings-पास-करना*/}` => `{/*passing-strings-with-quotes*/}`

do not translate these texts which are part of title in curly braces, those are used for ref links.

Update this for whole file